db-resources/MongoDB/index.js

```diff
@@ -0,0 +1,93 @@
+// Connecting to MongoDB without Mongoose ORM
+const { MongoClient } = require('mongodb'), format = require('util').format;
+const { getRandomArbitrary, getRandomWhole, scoreRound, getRandomPcnt, generateRandomCountry, getProductId } = require('./seedHelper')
+const { performance } = require('perf_hooks')
+
+MongoClient.connect('mongodb://127.0.0.1:27017', { useNewUrlParser: true }, (err, client) => {
+  if (err) {
+    console.log('error', err)
+    client.close();
+  } else {
+    console.log('Connected');
+    const db = client.db('newegg');
+    const product = db.collection('product')
+    product.drop();
+    batchInsert(product)
+      .then(() => {
+        console.log('database succeeded')
+        client.close()
+      })
+      .catch(() => {
+        console.log('database not succeeded')
+      })
```

Comment:
Here you use a Promise. 

Perhaps stick with one style over the other.  

Otherwise, this code is fine.

Reply:
Also, you close it when it completes successfully, but fail to close it if it fails to insert.

Reply:
Yes, please go over this part with me tomorrow.